ui/imports/shared/views/ProfileDialogView.qml

```diff
@@ -599,22 +605,24 @@ Pane {
 
                 // Profile Showcase
                 ProfileShowcaseView {
+                    id: showcaseView
+                    
                     Layout.fillWidth: true
                     Layout.topMargin: -column.spacing
                     Layout.preferredHeight: 300
 
                     currentTabIndex: showcaseTabBar.currentIndex
-                    publicKey: root.publicKey
                     mainDisplayName: d.mainDisplayName
                     readOnly: root.readOnly
-                    profileStore: root.profileStore
-                    walletStore: root.walletStore
-                    networkConnectionStore: root.networkConnectionStore
                     
-                    livePreview: root.dirty
-                    livePreviewValues: root.dirtyValues
+                    communitiesModel: root.showcaseCommunitiesModel
+                    accountsModel: root.showcaseAccountsModel
+                    collectiblesModel: root.showcaseCollectiblesModel
+                    // socialLinksModel: root.showcaseSocialLinksModel
+                    // assetsModel: root.showcaseAssetsModel
 
                     onCloseRequested: root.closeRequested()
+                    onCopyToClipboard: root.profileStore.copyToClipboard(text)
```

Comment:
I remember we have an`Utils.copyToClipboard` method. Would it be better here so we don't need a copy to clipboard method on each store??

Reply:
good point! Will have a look